nifi-registry-web-ui/src/main/webapp/services/nf-registry.service.js

```diff
@@ -37,8 +37,14 @@ var rxjs = require('rxjs/Observable');
 function NfRegistryService(nfRegistryApi, nfStorage, tdDataTableService, router, fdsDialogService, fdsSnackBarService) {
     var self = this;
     this.registry = {
-        name: "NiFi Registry"
+        name: "NiFi Registry",
+        // Config is updated later by calling the /config API.
+        config: {}
     };
+    nfRegistryApi.getRegistryConfig().subscribe(function (registryConfig) {
```

Comment:
Please move this call to getRegistryConfig  to https://github.com/apache/nifi-registry/blob/master/nifi-registry-web-ui/src/main/webapp/nf-registry.js#L56